.github/workflows/commit-validation.yml

```
@@ -49,21 +49,36 @@ jobs:
   tests-linux:
-    needs: build
+    needs: [build, test-matrix]
     strategy:
       fail-fast: false
       matrix:
-        node-version: [18, 20]
+        node-version: ${{ fromJSON(needs.test-matrix.outputs.node-versions) }}
     uses: ./.github/workflows/tests-linux.yml
     with:
       node-version: ${{matrix.node-version}}
+      base-node-version: ${{ needs.test-matrix.outbputs.base-node-version }}
```

actionlint (1.7.7)

input "base-node-version" is typed as number by reusable workflow "./.github/workflows/tests-linux.yml". string value cannot be assigned

(expression)